src/sinks/gcp/chronicle_unstructured.rs

```diff
@@ -127,6 +127,10 @@ pub struct ChronicleUnstructuredConfig {
     #[configurable(metadata(docs::examples = "c8c65bfa-5f2c-42d4-9189-64bb7b939f2c"))]
     pub customer_id: String,
 
+    /// User-configured environment namespace to identify the data domain the logs originated from.
+    #[configurable(metadata(docs::examples = "production"))]
+    pub namespace: String,
```

Comment:
This should be optional so it doesn't affect users that are currently not configuring a namespace...

```suggestion
    pub namespace: Option<String>,
```